stories/card/card.stories.js

```diff
@@ -411,10 +397,11 @@ export const listCard = () => `
 </div>
 `;
 
+listCard.storyName = 'List card';
 listCard.parameters = {
     docs: {
         iframeHeight: 400,
-        storyDescription: 'The Header and the Content sections follow the specification described in the Card Anatomy. The Content can accommodate different list types. All controls placed inside should behave natively.'
+        storyDescription: 'A card can display various types of lists. To display a list, add <code>fd-list</code> to the content element. All components placed inside should behave natively.'
```

Comment:
"To display a list, add <code>fd-list</code> to the content element." shouldn't be there as it might confuse the user. Adding <code>fd-list</code> will not be enough to render a list so it's better without this sentence.

Reply:
Is there anyway to make this description more useful to the user? It seems flat to me now. What about referencing the List component page? Like...

"To display a list in the content area, see **Standard List**." (and then link to the page)

Reply:
You can add the link. In my opinion with and without will be good enough. I let you decide if you want to add it :)